src/Ryujinx.Graphics.Vulkan/PipelineDynamicState.cs

```diff
@@ -60,88 +107,258 @@ public void SetDepthBias(float slopeFactor, float constantFactor, float clamp)
             _dirty |= DirtyFlags.DepthBias;
         }
 
+        public void SetDepthBiasEnable(bool enable)
+        {
+            _depthBiasEnable = enable;
+            _dirty |= DirtyFlags.DepthBiasEnable;
+        }
+
         public void SetScissor(int index, Rect2D scissor)
         {
             _scissors[index] = scissor;
-
             _dirty |= DirtyFlags.Scissor;
         }
 
-        public void SetStencilMasks(
-            uint backCompareMask,
-            uint backWriteMask,
-            uint backReference,
-            uint frontCompareMask,
-            uint frontWriteMask,
-            uint frontReference)
+        public void SetDepthTestBool(bool testEnable, bool writeEnable)
+        {
+            DepthTestEnable = testEnable;
+            DepthWriteEnable = writeEnable;
+            _dirty |= DirtyFlags.DepthTestBool;
+        }
+
+        public void SetDepthTestCompareOp(CompareOp depthTestOp)
+        {
+            _depthCompareOp = depthTestOp;
+            _dirty |= DirtyFlags.DepthTestCompareOp;
+        }
+
+        public void SetStencilTestandOp(StencilOp backFailOp, StencilOp backPassOp, StencilOp backDepthFailOp,
```

Comment:
```suggestion
        public void SetStencilTestandOp(
        StencilOp backFailOp,
        StencilOp backPassOp,
        StencilOp backDepthFailOp,
```
Would be nice to separate the arguments like this for consistency.